src/ifp/README.md

```diff
@@ -52,30 +52,26 @@ die =  ( 0, 0 )
 
 ```tcl
 initialize_floorplan
+  (-die_area {llx lly urx ury} -core_area {llx lly urx ury}) | (-utilization util [-aspect_ratio ratio] [-core_space (space | {bottom top left right})])
```

Comment:
`[-core_space` should drop the [ as it isn't optional.

Reply:
-core_space is optional in current code. When no stecipied that core space is 0. Should I make it mandatory?
```
    if { [info exists keys(-core_space)] } {
      set core_sp $keys(-core_space)
      if { [llength $core_sp] == 1 } {
        sta::check_positive_float "-core_space" $core_sp
        set core_sp_bottom $core_sp
        set core_sp_top $core_sp
        set core_sp_left $core_sp
        set core_sp_right $core_sp
      } elseif { [llength $core_sp] == 4 } {
        lassign $core_sp core_sp_bottom core_sp_top core_sp_left core_sp_right
        sta::check_positive_float "-core_space" $core_sp_bottom
        sta::check_positive_float "-core_space" $core_sp_top
        sta::check_positive_float "-core_space" $core_sp_left
        sta::check_positive_float "-core_space" $core_sp_right
      } else {
        utl::error IFP 13 "-core_space is either a list of 4 margins or one value for all margins."
      }
    } else {
      set core_sp_bottom 0.0
      set core_sp_top 0.0
      set core_sp_left 0.0
      set core_sp_right 0.0
    }
```

Reply:
Yes I think it should be mandatory in the same with -core_area is.

Reply:
Ok. Fixed.